src/test/cypress/support/pageobjects/AssessmentDashboard.ts

```diff
@@ -0,0 +1,31 @@
+export class AssessmentDashboard {
+    openExerciseDashboard() {
+        cy.get('[jhitranslate="entity.action.exerciseDashboard"]').should('be.visible').click();
+    }
+
+    confirmInstruction() {
+        cy.contains(/I have read and understood the instructions|Start participating in the exercise/).click();
```

Comment:
Matching strings should be the last resort. In my text exercise assessment PR I used [this selector](https://github.com/ls1intum/Artemis/blob/778675b0f629bfe67100229484116e4a36ab38bb/src/test/cypress/support/pageobjects/assessment/ExerciseAssessmentDashboardPage.ts#L6). Might be different for exam assessments though

Reply:
found a better one.